test/framework/easystack.py

```diff
@@ -71,10 +71,11 @@ def test_easystack_basic(self):
         self.assertEqual(sorted(ec_fns), sorted(expected))
         self.assertEqual(opts, {})
 
-    def test_easystack_easyconfigs(self):
-        """Test for easystack file using 'easyconfigs' key."""
+
+    def test_easystack_basic_dict(self):
```

Comment:
Check if the format in `test_easystack_basic_dict.yaml` _should_ be valid. If so, we should change the parser to make this pass. If not, we need to change this test and check for the failure

Reply:
fixed in a5ce7d2